plugins/src/main/java/com/google/firebase/gradle/plugins/FirebaseLibraryExtension.kt

```diff
@@ -69,6 +69,22 @@ constructor(val project: Project, val type: LibraryType) {
    */
   abstract val publishJavadoc: Property<Boolean>
 
+  /**
+   * Only publish Kotlindocs instead of also publishing Javadocs for this library.
+   *
+   * This property is only effective when `publishJavadoc` is `true`.
+   *
+   * Defaults to `false`.
+   *
+   * ```sh
```

Comment:
What happens if only this flag is set, but not the `publishJavadoc` one?

Reply:
As documented in the comment, nothing.